nuxeo-page-provider-display-behavior.html

```diff
@@ -417,8 +418,11 @@
             delete this.nxProvider.params[e.detail.filterBy];
             this.fetch();
           } else if (e.detail.value.length > 0) {
-            if (e.detail.filterBy === 'title' || e.detail.filterBy === 'ecm_fulltext') {
+            if ((e.detail.filterBy === 'title' || e.detail.filterBy === 'ecm_fulltext') && !e.detail.filterExpression) {
```

Comment:
We could drop the `!e.detail.filterExpression` if we did the `if (e.detail.filterExpression)` first. I would also be easier to read.

Reply:
Well spotted, thanks!